src/coreclr/tools/Common/Compiler/CompilerTypeSystemContext.cs

```diff
@@ -54,7 +54,7 @@ protected override ModuleData CreateValueFromKey(EcmaModule key)
                 return null;
             }
         }
-        private readonly ModuleHashtable _moduleHashtable = new ModuleHashtable();
```

Comment:
I believe this change isn't necessary now. Please change back to private.

Reply:
`_moduleHashtable` is used outside of containing type in `InheritOpenModules`.

Reply:
Maybe I'm not seeing this right in the Gihub UI but from what I see, both _moduleHashtable and InheritOpenModules are members of the CompilerTypeSystemContext class - therefore _moduleHashtable will be accessible from InheritOpenModules even if it's private.

Reply:
It works indeed. Thanks!